src/plots/font_weight.js

```diff
@@ -0,0 +1,13 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+// TODO make this a plot attribute?
+module.exports.fontWeight = 'normal';
```

Comment:
This seems silly to have in its own file but was another one of those couple things that required a giant circular require of top-level `Plots` by some deeply nested component.

Reply:
This is only used in one place, `titles` - everywhere else we just implicitly inherit `font-weight: normal`.

One *can* set `font-weight: bold` or something on `gd` or a parent, and have that inherited by text inside the plot. We could argue about whether that's a reasonable thing to allow (in particular, exports would NOT inherit this bolding) but it seems silly to ONLY explicitly remove this from the title. I'd 🔪  this file AND the place it's used in `titles/index.js`. Later if we decide this is important we can apply normal weight to our whole `<svg>` elements or something.

Reply:
By the way, that font weight key has to be exposed on `Plots` for [this](https://github.com/plotly/streambed/blob/master/shelly/plotlyjs/static/plotlyjs/src/styles/open_office_2015.js#L12).

Reply:
> By the way, that font weight key has to be exposed on `Plots` for [this](https://github.com/plotly/streambed/blob/master/shelly/plotlyjs/static/plotlyjs/src/styles/open_office_2015.js#L12).

Ah, crazy - as it stands now this PR will break that, but now I get why this was in there in the first place. So it's really just to allow us to make titles bold by default. Then lets have this an attribute of the `Titles` object, ie `Titles.fontWeight`, so it can be overridden by saying `var Titles = require('./components/titles'); Titles.fontWeight = 900;`